recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
I believe @msarahan updated the quoting of this? I remember discussing this before somewhere.